youtube_dl/extractor/livestreamfails.py

```diff
@@ -0,0 +1,49 @@
+from .common import InfoExtractor
+import json
+import time
+import calendar
+
+
+class LivestreamfailsIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?livestreamfails\.com/clip/(?P<id>[0-9]+)'
+    _TESTS = [{
+        'url': 'https://livestreamfails.com/clip/139200',
+        'md5': '8a03aea1a46e94a05af6410337463102',
+        'info_dict': {
+            'id': '139200',
+            'ext': 'mp4',
+            'display_id': 'ConcernedLitigiousSalmonPeteZaroll-O8yo9W2L8OZEKhV2',
+            'title': 'Streamer jumps off a trampoline at full speed',
+            'creator': 'paradeev1ch',
+            'thumbnail': 'https://livestreamfails-image-prod.b-cdn.net/image/3877b1d38db083fa25c82685bbaf645637e575ea.png',
+            'timestamp': 1656271785,
+            'upload_date': '20220626',
+        }
+    }]
+
+    def _real_extract(self, url):
+        result = {}
+        result['id'] = self._match_id(url)
+
+        # https://livestreamfails.com/clip/id uses https://api.livestreamfails.com/clip/ to fetch the video metadata
+        # Use the same endpoint here to avoid loading and parsing the provided page (which requires JS)
+        apiResponse = json.loads(self._download_webpage('https://api.livestreamfails.com/clip/' + result['id'], result['id']))
+
+        # Twitch ID of clip
+        result['display_id'] = apiResponse.get('sourceId')
```

Comment:
This should fail if the data is not available, or default to `IE._generic_id()`:
```suggestion
        result['display_id'] = apiResponse.get['sourceId']
```
After this, `apiResponse` is known to be a dict that you can `.get()` from without crashing.

Reply:
I would not value `sourceId` that highly since it links back to Twitch and therefore doesn't affect the downloaded video. Is it really worth breaking the entire extractor for what is arguably a niche situation?

Reply:
Right, that's fine for `display_id`.